lib/modules/config.js

```diff
@@ -12,6 +12,7 @@ class Config {
       files: ["README.md"],
       imageSize: 100,
       commit: false,
+      commitTag: "docs",
```

Comment:
I think it's usually referred to as commit scope (see https://www.conventionalcommits.org/en/v1.0.0/). `commitTag` might be confusing, as `tag` as a different meaning in the context of git

Reply:
yes, you are correct, it's commit type 👍🏼 sorry I got it wrong initially. Thanks for looking it up